README.md

```diff
@@ -126,6 +126,12 @@ spec:
 
 # Changelog
 
+## 2.0.3
+
+* bump `common` dependency to `3.44.0`
+* bump `common-utils` dependency to `0.0.3`
+* make `batchByGroup` setting default to `false`
+
 ## 2.0.2
 
 * add channel flush timeout
```

Comment:
Please change `add support for session groups` to `prepared architecture for support session groups`